external-notification-service.openApi.yaml

```diff
@@ -0,0 +1,48 @@
+openapi: '3.0.1'
+info:
+  title: UKHO External Notification Service APIs
+  description: "This is a light weight open api spec for UKHO External Notification Service. One of its main purpose is to allow importing ENS APIs in APIM."
+  version: '0.0.1'
+  contact:
+    email: martin.rock-evans@ukho.gov.uk
```

Comment:
Not sure if this should be martin or some more general inbox in UKHO.

Reply:
FSS & ESS specs have his email address. Can you share a general email address if want it here.

Reply:
Updated.